api/proxy/config/enabled_apis/cli.go

```diff
@@ -0,0 +1,39 @@
+package enabled_apis
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	EnabledAPIsFlagName = "apis.enabled"
+)
+
+func withEnvPrefix(envPrefix, s string) []string {
+	return []string{envPrefix + "_" + s}
+}
+
+func ReadEnabledAPIs(ctx *cli.Context) *EnabledAPIs {
+	enabledAPIStrings := ctx.StringSlice(EnabledAPIsFlagName)
+
+	apis, err := StringsToEnabledAPIs(enabledAPIStrings)
+	if err != nil {
+		panic(err)
+	}
+
+	return apis
+}
+
+func CLIFlags(category string, envPrefix string) []cli.Flag {
+	return []cli.Flag{&cli.StringSliceFlag{
+		Name: EnabledAPIsFlagName,
+		Usage: fmt.Sprintf("Which proxy application APIs to enable. supported options are "+
+			"%s, %s, %s, %s, %s, %s", Admin.ToString(), StandardCommitment.ToString(),
+			OpGenericCommitment.ToString(), OpKeccakCommitment.ToString(),
+			ArbCustomDAServer.ToString(), MetricsServer.ToString()),
+		Value:    cli.NewStringSlice(),
```

Comment:
by default no api is turned on? why not make it required instead?

Reply:
[590c2bf](https://github.com/Layr-Labs/eigenda/pull/1985/commits/590c2bf6949db20e430762e6210ed43e34919af5)

Reply:
initially I didn't make it required since we don't actually have any required CLI flags for the proxy but enforce a notion of "required" when doing expressional config invariance

Reply:
wtf lol never realized. This is kind of dumb. I'm sure there's a way to disable global flags for a subcommand no?
Otherwise we might want to make starting the proxy another subcommand with its own flags.. and then make that subcommand default (also not sure if this is possible...)

Anyways all of this is doesnt need to happen in this PR of course.

Reply:
will set `Flags` to false

Reply:
25c7d7c3b5e56ed2c6c370384c510e108e57ff1b